gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/jackson/ser/api/ApiSerializer.java

```diff
@@ -107,12 +108,19 @@ public void serialize(ApiEntity apiEntity, JsonGenerator jsonGenerator, Serializ
         if (apiEntity.getPicture() != null) {
             jsonGenerator.writeObjectField("picture", apiEntity.getPicture());
         }
-        if (apiEntity.getPaths() != null) {
-            jsonGenerator.writeObjectFieldStart("paths");
-            for (Map.Entry<String, List<Rule>> entry : apiEntity.getPaths().entrySet()) {
-                jsonGenerator.writeObjectField(entry.getKey(), entry.getValue());
+
+        if (DefinitionVersion.V1.getLabel().equals(apiEntity.getGraviteeDefinitionVersion())) {
+            if (apiEntity.getPaths() != null) {
+                jsonGenerator.writeObjectFieldStart("paths");
+                for (Map.Entry<String, List<Rule>> entry : apiEntity.getPaths().entrySet()) {
+                    jsonGenerator.writeObjectField(entry.getKey(), entry.getValue());
+                }
+                jsonGenerator.writeEndObject();
+            }
+        } else if (DefinitionVersion.V2.getLabel().equals(apiEntity.getGraviteeDefinitionVersion())) {
```

Comment:
Should we better use a `else` and don't specify V2 version, as flows will have to be serialized for V4 also ?